### PR TITLE
Add docs for new ROOT_SIZE acceptable arguments.

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by jlebon** [comment]
Probably needs a `udevadm settle` in there.

**Comment by jlebon** [comment]
bot, retest this please

**Comment by rhvgoyal** [comment]
@dustymabe we need to update comments in container-storage-setup.conf as well. Also please modify code to not use gloabl `SIZE_ARG`. Instead use a local variable. Say ` local root_size_args`

**Comment by dustymabe** [comment]
@rhvgoyal - i think both requests have been included - see updated code

**Comment by dustymabe** [comment]
bot, retest this please

File: container-storage-setup.sh [hunk 1]
```diff
@@ -643,14 +643,15 @@ grow_root_lv_fs() {
     # Allow user to pass in an argument that could be provided
     # to -L (like 10G or +5G) or an argument that could be passed
     # to -l (like 80%FREE). Switch based on if '%' is in string.
+    local local root_size_arg
```
Comment (rhvgoyal) [inline container-storage-setup.sh]: You don't have to say `local` twice.

**Comment by dustymabe** [comment]
> You don't have to say local twice.

oops. fixed

**Comment by rhvgoyal** [comment]
LGTM

Though some test are breaking on my local machine. But I feel its a separate issue. Will debug tomorrow.

**Comment by rhvgoyal** [comment]
@rh-atomic-bot r+ 4d3c6fc

**Pull request closed without merging**